bluebottle/payments/services.py

```diff
@@ -1,6 +1,6 @@
 from bluebottle.payments.exception import PaymentException
 import re
-from django.conf import settings
+from bluebottle.clients import properties
```

Comment:
@iivvoo do you mean bluebottle.clients.properties instead of bluebottle.clients.settings?